frontend/src/components/InterviewPage.js

```diff
@@ -25,8 +24,8 @@ const InterviewPage = () => {
     const username = location.state.username
 
     useEffect(() => {
-        socket.emit('join_room', { username, room: interviewId}); // Connect user to chat room
-        collabSocket.emit("JOIN", { roomId: interviewId })
+        chatSocket.emit('join', { username, room: interviewId }) // Connect user to chat socket room
+        collabSocket.emit("JOIN", { roomId: interviewId })       // Connect user to collab socket room 
```

Comment:
Will it be confusing if the only differentiating factor is capital letters?

Reply:
Not planning to differentiate them. "join" will mean to join that particular socket. I have asked Herrick to change to "join" as well and standardize his routes to non-capital letters.

Reply:
Maybe should standardize the naming of the parameters as well. chatSocket is using { room : interviewId } but collabSocket is using { roomId : interviewId }

Reply:
@Herrekt can acknowledge and resolve this convo

Reply:
Understood. Will do the required changes in the collab service test/ci or dockerizing PR.